### PR TITLE
fix(TBD-13195): add missing libs for synapse

### PR DESCRIPTION
**Pull request by AlixMetivier**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by firefryy — DISMISSED** [review]
lgtm

File: main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml [hunk 1]
```diff
@@ -476,12 +476,12 @@
         
         <!-- Bigdata launcher group -->
         <libraryNeeded context="plugin:org.talend.hadoop.distribution.synapse"
-          id="talend-bigdata-launcher-core-2.1.2.jar" name="talend-bigdata-launcher-core-2.1.2.jar" 
-          mvn_uri="mvn:org.talend.bigdata.libs/talend-bigdata-launcher-core/2.1.2"/>
+          id="talend-bigdata-launcher-core-2.3.0.jar" name="talend-bigdata-launcher-core-2.3.0.jar"
```
Comment (sponomarova) [inline main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml:479]: correct version of synapse launcher is 2.4.3.

File: main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml [hunk 2]
```diff
@@ -476,12 +476,12 @@
...
         <libraryNeeded context="plugin:org.talend.hadoop.distribution.synapse"
-          id="talend-bigdata-launcher-core-2.1.2.jar" name="talend-bigdata-launcher-core-2.1.2.jar" 
-          mvn_uri="mvn:org.talend.bigdata.libs/talend-bigdata-launcher-core/2.1.2"/>
+          id="talend-bigdata-launcher-core-2.3.0.jar" name="talend-bigdata-launcher-core-2.3.0.jar" 
+          mvn_uri="mvn:org.talend.bigdata.libs/talend-bigdata-launcher-core/2.3.0"/>
         
         <libraryNeeded context="plugin:org.talend.hadoop.distribution.synapse"
-          id="talend-bigdata-launcher-synapse-2.1.2.jar" name="talend-bigdata-launcher-synapse-2.1.2.jar" 
-          mvn_uri="mvn:org.talend.bigdata.libs/talend-bigdata-launcher-synapse/2.1.2"/>
+          id="talend-bigdata-launcher-synapse-2.3.0.jar" name="talend-bigdata-launcher-synapse-2.3.0.jar"
```
Comment (sponomarova) [inline main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml:483]: version

File: main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml [hunk 3]
```diff
@@ -620,8 +621,8 @@
         
          <!-- Bigdata launcher group -->
         <libraryNeededGroup description="Bigdata launcher libraries for synapse" id="BIGDATA-LAUNCHER-LIB-SYNAPSE" name="BIGDATA-LAUNCHER-LIB-SYNAPSE">
-        	<library id="talend-bigdata-launcher-synapse-2.3.0"/>
-            <library id="talend-bigdata-launcher-core-2.3.0"/>
+        	<library id="talend-bigdata-launcher-synapse-2.3.0.jar"/>
```
Comment (sponomarova) [inline main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml:624]: version

**Review by sponomarova — CHANGES_REQUESTED** [review]
Please, change a version of synapse launcher to 2.4.3

File: main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml (hunk 2 above)
Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml:483]: Should be talend-bigdata-launcher-synapse-2.4.3.jar

Comment (AlixMetivier) [reply main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml:483]: its talend-bigdata-launcher-synapse-2.3.0 on maintenance/7.3, should it be updated there as well ?

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.synapse/plugin.xml]: It has to be updated for maintenance/7.3 as well. 
Now bigdata-launcher-synapse is taken from dependencies in tSparkConfiguration:
https://github.com/Talend/tbd-studio-ee/blob/8d482b873860936f176fc98cafb2b1dcb4ea7c20/main/plugins/org.talend.designer.components.sparkprovider/components/tSparkConfiguration/tSparkConfiguration_java.xml#L1581

**Review by sponomarova — APPROVED** [review]

**Pull request merged**